validator_client/src/config.rs

```diff
@@ -32,8 +32,8 @@ pub struct Config {
     pub delete_lockfiles: bool,
     /// If true, don't scan the validators dir for new keystores.
     pub disable_auto_discover: bool,
-    /// If true, don't re-register existing validators in definitions.yml for slashing protection.
-    pub strict_slashing_protection: bool,
+    /// If true, re-register existing validators in definitions.yml for slashing protection.
```

Comment:
Does this also register validators that didn't exist originally? Just checking this is worded accurately.

Reply:
It's a little bit clunky, but I think it will register all validators from the validator definitions YAML, I'll earmark it for improvement in a subsequent PR, seeing as we're tight for time